pkg/plugins/manager/pipeline/initialization/initialization.go

```diff
@@ -54,7 +54,7 @@ func (i *Initialize) Initialize(ctx context.Context, ps []*plugins.Plugin) ([]*p
 			ip, err = init(ctx, p)
 			if err != nil {
 				stepFailed = true
-				i.log.Error("Could not initialize plugin", "pluginId", p.ID, "err", err)
+				i.log.Error("Could not initialize plugin", "pluginID", p.ID, "err", err)
```

Comment:
Nit. Prefer using camelCase when logging https://github.com/grafana/grafana/blob/main/contribute/backend/instrumentation.md#naming-conventions

```suggestion
				i.log.Error("Could not initialize plugin", "pluginId", p.ID, "error", err)
```

Reply:
Had no idea! Have gone ahead and tried fix across the entire `pkg/plugins`!